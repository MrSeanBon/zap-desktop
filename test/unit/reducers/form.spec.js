import formReducer, { SET_FORM_TYPE } from 'reducers/form'

describe('reducers', () => {
  describe('formReducer', () => {
    it('should handle initial state', () => {
      expect(formReducer(undefined, {})).toMatchSnapshot()
    })

    it('should have SET_FORM_TYPE', () => {
      expect(SET_FORM_TYPE).toEqual('SET_FORM_TYPE')
    })

    it('should correctly setFormType', () => {
      expect(formReducer(undefined, { type: SET_FORM_TYPE, formType: 'FOO' })).toMatchSnapshot()
    })
  })
})
