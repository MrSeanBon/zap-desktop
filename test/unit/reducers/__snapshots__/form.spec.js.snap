// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`reducers formReducer should correctly setFormType 1`] = `
Object {
  "formType": "FOO",
}
`;

exports[`reducers formReducer should handle initial state 1`] = `
Object {
  "formType": null,
}
`;
